webshare/share-url-noscheme-manual.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Manual Test: Share with URL without a scheme</title>
```

Comment:
I was cargo-culting from other tests in WPT (specifically, geolocation-API) in which every test is either manual or https (but not both). I see other tests with the suffix "`-manual.https.html`".

Should these all be renamed?